packages/delivery-fetch/delivery.js

```diff
@@ -0,0 +1,44 @@
+import payload from '@bugsnag/core/lib/json-payload'
+
+const delivery = (client) => ({
+  sendEvent: (event, cb = () => {}) => {
+    const url = client._config.endpoints.notify
+
+    global.fetch(url, {
```

Comment:
Other plugins that use globals allow passing a replacement when they are imported, e.g. in `plugin-window-onerror`:

https://github.com/bugsnag/bugsnag-js/blob/0599faeb4019e571bfd7111c2663b5ddcbb78901/packages/plugin-window-onerror/onerror.js#L5

So we should pass `fetch` to this function with a default value of `fetch` (i.e. the global)